delius-perf/sub-projects/cloudplatform.tfvars

```diff
@@ -4,4 +4,4 @@
 # See https://dsdmoj.atlassian.net/wiki/spaces/DAM/pages/1850507436/Probation+in+Court+services+access+NewTech+ElasticSearch+cluster
 #------------------------------------
 
-cloudplatform_offender_search_role_name = "offender-search-elastic-search-delius-perf"
\ No newline at end of file
+cloudplatform_offender_search_role_name = "offender-search-elastic-search-delius-pre-prod"
```

Comment:
there is no perf role, nor to we need one right now since there is no connection between any service in cloud platform in perf (or indeed any Delius environment )